common/constants.js

```diff
@@ -0,0 +1 @@
+module.exports = { nameMapFile: 'names-from-templates.json' };
```

Comment:
Following up on our conversation [here](https://github.com/benblank/ember-first-class-component-templates-migrator/pull/6/files#r1036033999), it would actually be better if this is just exported from something specific to the Stage 2 file: that makes the contract clear. It's not so much "common" as it is "Stage 2 publishes this, so Stage 4 needs to know what Stage 2 did".